File: SocketChat/Context/Entities/Hub.cs
public class Hub
{
	public long Id { get; set; }
	public string Hubname { get; set; }
	public bool Status { get; set; }

	public DateTime? AddedDate { get; set; }
	public DateTime? UpdatedDate { get; set; }
	public DateTime? DeletedDate { get; set; } // Nullable to represent optional dates

	public bool IsActive { get; set; }
}

File: SocketChat/SocketChat.Model/Entities/Hub.vb
Imports System.Security.Principal

Public Class Hub : Implements IEntity(Of Long)
    Public Property Id As Long Implements IEntity(Of Long).Id
    Public Property Hubname As String
    Public Property Status As Boolean


    Public Property AddedDate As Date? Implements IEntity(Of Long).AddedDate
    Public Property UpdatedDate As Date? Implements IEntity(Of Long).UpdatedDate
    Public Property DeletedDate As Date? Implements IEntity(Of Long).DeletedDate

    Public Property IsActive As Boolean Implements IEntity(Of Long).IsActive
End Class
